app/styles/ui/toolbar/_dropdown.scss

```diff
@@ -6,6 +6,9 @@
   display: flex;
   flex-direction: row;
 
+  // Fill the available height of the parent
+  flex-grow: 1;
+
```

Comment:
Fix for the `.branch-button` not filling the available height of its parent. Indirectly, it also makes the same rule on `.sidebar-section > :last-child` redundant so it can be removed.